docs/releasing.md

```diff
@@ -2,41 +2,83 @@
+   Most importantly, the log will explain how the version number would be bumped, based on new commits to `main`. 
 
-   - Determine the new version based on new commit messages
-   - Generate a new entry in the changelog with the version, release notes, and today's date.
-   - Commit all of the above changes with the message `chore(release): <version>`
+3. Create a release branch for the new version to be released, where `{new.version.number}` might be `2.4.1`, for example.
+   
+   ```
+   git checkout -b release-{new.version.number} 
+   ```
 
-3. Push changes to a new branch following the naming pattern: `release-<version>`
+4. On the newly created release branch, allow `standard-version` to create the release
+   
+   ```
+   yarn release
+   ```
+   
+   Without using `--dry-run`, `standard-version` will commit the following changes with the message `chore(release): {new.version.number}`:
+   - Bump the version in `package.json`
+   - Generate a new entry in `CHANGELOG.md` with the version, release notes, and today's date.
 
-   - For example: `git checkout -b release-1.1.0`
+5. Push the changes to Github
+   
+   ```
+   git push -u origin release-{new.version.number}
+   ```
 
-4. Open a PR for the release branch against `main`, with the changelog generated by the previous step included in the PR description.
+6. Open a PR for the release branch against `main`
 
    - PR title should be `chore(release): <version>`
+   - PR should have the github label [type: release](https://github.com/trussworks/react-uswds/issues?q=label%3A%22type%3A+release%22+).
+   - Paste the the changelog generated by `standard-version` in the PR description/body.
    - Ask for approvals from stakeholders, perform testing on applications, etc.
+      - It can be helpful to reviewers to provide a diff of all the files that have changed since the last release. To do this, add a compare link as a comment to the PR, or in the description/body:
+      `https://github.com/trussworks/react-uswds/compare/{previous.release.tag}...release-{new.version.number}`
 
    ![image](./release_PR.png)
 
-5. Once the release PR is approved and merged, complete the release by publishing the new version:
-      -  Create a new [**release**](https://github.com/trussworks/react-uswds/releases) on Github, pointed at the merge commit of the release PR. 
-         - Look for the `Draft a new release` button.
-         - Set the tag to the version number.  
-         - Title your release with the version number. Use the same release notes as the release PR.
-         - Point the release tag at the release merge commit by selecting `Recent Commits` from the `Target` dropdown in the Github UI. You should find the `chore(release): ...` commit at or near the top.
+7. Once the release PR is approved and merged, complete the release by publishing the new version
```

Comment:
One thing @rogeruiz and I talked about as well was tagging the release on the release branch, rather than after merge to main. 

Doing so would prevent all issues with main receiving changes after a release is started (dependabot or otherwise). 

The package release action would need to be updated to act on tag/release creation, but I don't think that's a huge deal.

Also happy to leave as is for now and file an issue as a suggestion in the name of incremental improvement.

Reply:
I would like to see to this filed as an issue (if the work isn't done yet) just to get this PR through. I really like that change its a good incremental improvement.